packages/ember-glimmer/tests/utils/helpers.js

```diff
@@ -1,13 +1,10 @@
-import assign from 'ember-metal/assign';
-import {
-  compile as compiler,
-  precompile as precompiler,
-  defaultCompileOptions,
-  template
-} from 'ember-glimmer-template-compiler';
 import { setupApplicationRegistry, setupEngineRegistry } from 'ember-glimmer/setup-registry';
 import { default as _buildOwner } from 'container/tests/test-helpers/build-owner';
 
+export {
+  compile,
+  precompile
+} from 'ember-glimmer-template-compiler';
```

Comment:
Should this be using `ember-glimmer-template-compiler/tests/utils/helpers` (seems like the `ember-htmlbars/tests/utils/helpers` version does use the `ember-htmlbars-template-compiler/tests/utils/helpers`).